### DISCUSSION TITLE
MDCKII-MDR1 transporter Papp

### DISCUSSION THREAD

**Discussion by RajFer** [Open-Systems-Pharmacology/Forum#1462] [How-Tos]
Hi, 
I am new to OSP. 
Great if somebody clarify me the following: 
I have a measured MDCKII-MDR1 transporter Papp value and struggling to how to enter this in PK-Sim. 
Tried "Transport Proteins" under "Transport and Excretion" tab but it requires Vamx and Km not the Permeability Coefficient or ER
values you would get in MDCK assays. 
Thanks very much in advance.

**Comment by RobinM92** [comment]
Hi @RajFer,

If you know the cell surface area of your experiments and have the Papp at different substrate concentrations, you could in theory calculate Vmax and Km by fitting the data to a michaelis menten equation yourself:

CLint (µl/min) ~ Papp (cm/s) * SA (cm²) * 60 (s/min) * 10^(3) (µL/cm^3)
V (nmole/min) ~ CLint  (µl/min) * [S] (µM) * 1000 (nmole/µmole) for each substrate concentration

Plot V vs [S], fit V = Vmax*[S]/(Km + [S])

The problem is that you then still need your transporter concentration in order to be able to use this value.
I know work was being done to find default values for these transporter concentrations for the standard cell lines, but am not aware of any results there yet. Perhaps you have measured them? 

See e.g. [https://github.com/Open-Systems-Pharmacology/Forum/discussions/830](here) 

But I am sure the experts here can weigh in much more than I can :)
